### DISCUSSION TITLE
Estimated Release Date for Final Version of 3.0.0?

### DISCUSSION THREAD

**Discussion by shusaan** [distribution/distribution#4526] [Q&A]
Hi there! I noticed that version 3.0.0-rc.1 has been released. I was wondering if there’s an estimated timeline for the release of the final version of 3.0.0? Any update would be greatly appreciated!

The current release candidate is using Alpine 3.20.3. However, Alpine 3.21 has been released, and I wanted to suggest considering an upgrade to the newer version for the final 3.0.0 release.

**Comment by milosgajdos** [comment]
No stable release is planned at the moment. I think we might cut rc-2 before EOY, but don't hold your breath.

Hopefully early 2025 we will cut a stable release, but again, no guarantees; at this point it's mostly me who does most of work in my free unpaid time, so update your expectations accordingly :-)

**Comment by vineetsingh5** [comment]
I can see RC2 is created 1 week ago. Is there any update regarding the final release timeline? 
We have this issue https://github.com/distribution/distribution/issues/4549 and want to upgrade to latest version
Thanks
